src/components/KBreadcrumbs/KBreadcrumbs.vue

```diff
@@ -102,9 +114,9 @@ export default {
   display: flex;
   -ms-flex-wrap: wrap;
   flex-wrap: wrap;
-  font-size: 15px;
-  font-weight: 600 !important;
-  line-height: 24px !important;
+  font-size: $kui-font-size-30;
+  font-weight: $kui-font-weight-medium;
+  line-height: $kui-line-height-40;
```

Comment:
We need to utilize the CSS variables here as well (all instances where you're using SCSS variables). Applies to the whole PR, but example here for this block:
```suggestion
  font-size: var(--kui-font-size-30, $kui-font-size-30);
  font-weight: var(--kui-font-weight-medium, $kui-font-weight-medium);
  line-height: var(--kui-line-height-40, $kui-line-height-40);
```